AngularJS/angularjs-quiz.md

```diff
@@ -1,199 +1,226 @@
 ## AngularJs LinkedIn Assessment Questions Answers (Left)
 
+
 #### Q1. What is the purpose of the ViewChild decorator in this component class?
 
+```javascript
```

Comment:
can please try to use angular instead of javascript which is more related to this quiz?

Reply:
No, better stay with JS. I show:
![image](https://user-images.githubusercontent.com/3178925/104051136-eff46a80-51e7-11eb-876c-f73b53c0d7e9.png)

Reply:
yes agree, with js look really better => resolved